shared/src/main/scala/minitest/runner/Framework.scala

```diff
@@ -18,7 +18,7 @@
 package minitest.runner
 
 import minitest.runner.Framework.ModuleFingerprint
-import sbt.testing.{Framework => BaseFramework, _}
+import sbt.testing.{Framework => BaseFramework, Runner => BaseRunner, _}
```

Comment:
Why?

Reply:
Name resolution changed, this is to hide the `sbt.testing.Running` and select the correct one bellow. In scala2 mode there is a helpful message that warns about the change.

Reply:
You can do:
```scala
import sbt.testing.{Framework => BaseFramework, Runner => _, _}
```

Reply:
Ok